apps/crunchy-postgres/workload/run_litmus_test.yml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: crunchy-postgres-loadgen 
```

Comment:
can you use generateName here to eliminate the usage of duplicate job names.

Reply:
@gprasath  sure i wll make the changes